<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StanzaEditorActivity</class>
 <widget class="QWidget" name="StanzaEditorActivity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Stanza</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditStanzaTitle"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLobe">
       <property name="text">
        <string>Select Lobe</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PoseMappingWidget" name="widgetPoseMapping" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PoseMappingWidget</class>
   <extends>QWidget</extends>
   <header location="global">pose/PoseMappingWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonLobe</sender>
   <signal>clicked()</signal>
   <receiver>StanzaEditorActivity</receiver>
   <slot>selectLobe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>StanzaEditorActivity</receiver>
   <slot>cance()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>StanzaEditorActivity</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectLobe()</slot>
  <slot>save()</slot>
  <slot>cance()</slot>
 </slots>
</ui>
